src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -344,7 +353,7 @@
            directory.
        2.  Runs the tests. -->
   <Target Name="GenerateTestExecutionScripts"
-          DependsOnTargets="DiscoverTestInputs;SetupTestProperties;MakeTestSpecificResourcesPriFile;MakeCommonResourcesPriFile">
```

Comment:
Why do we don't depend on all the supplemental data that is added in DiscoverTestInputs target?

Reply:
The supplemental test data and the configuration files are copied as part of the build target. To run the test target the assembly needs to be built. No need to have the depends on here as as that target is part of the build.